src/consensus/aft/raft.h

```diff
@@ -646,6 +697,20 @@ namespace aft
             break;
           }
 
+          case raft_node_caught_up:
+          {
+            NodeCaughtUpMsg r =
+              channels->template recv_authenticated_with_load<NodeCaughtUpMsg>(
+                from, data, size);
+
+            aee = std::make_unique<NodeCatchUpCallback>(
+              from,
+              configuration_tracker,
+              TxID({r.view, r.seqno}),
+              TxID({state->current_view, state->last_idx}));
```

Comment:
This is a pair of seqno/view but I don't think this is a valid `TxID`: `last_idx` could have been committed in the previous view but we're not at `current_view`.